docs/content/release-notes/flink-1.13.md

```diff
@@ -0,0 +1,227 @@
+"AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+KIND, either express or implied.  See the License for the
+specific language governing permissions and limitations
+under the License.
+-->
+
+# Release Notes - Flink 1.13
+
+These release notes discuss important aspects, such as configuration, behavior, or dependencies,
+that changed between Flink 1.12 and Flink 1.13. Please read these notes carefully if you are
+planning to upgrade your Flink version to 1.13.
+
+### Failover
+
+#### Remove state.backend.async option.
+
+##### [FLINK-21935](https://issues.apache.org/jira/browse/FLINK-21935)
+
+The `state.backend.async` option is deprecated. Snapshots are always asynchronous now (as they were
+by default before) and there is no option to configure a synchronous snapshot any more.
+
+The constructors of `FsStateBackend` and `MemoryStateBackend` that take a flag for sync/async
+snapshots are kept for API compatibility, but the flags are ignored now.
+
+#### Unify binary format for Keyed State savepoints
+
+##### [FLINK-20976](https://issues.apache.org/jira/browse/FLINK-20976)
+
+Flink’s savepoint binary format is unified across all state backends. That means you can take a
+savepoint with one state backend and then restore it using another.
+
+If you want to switch the state backend you should first upgrade your Flink version to 1.13, then
+take a savepoint with the new version, and only after that, you can restore it with a different
+state backend.
+
+#### FailureRateRestartBackoffTimeStrategy allows one less restart than configured
+
+##### [FLINK-20752](https://issues.apache.org/jira/browse/FLINK-20752)
+
+The Failure Rate Restart Strategy was allowing 1 less restart per interval than configured. Users
+wishing to keep the current behavior should reduce the maximum number of allowed failures per
+interval by 1.
+
+#### Support rescaling for Unaligned Checkpoints
+
+##### [FLINK-17979](https://issues.apache.org/jira/browse/FLINK-17979)
+
+While recovering from unaligned checkpoints, users can now change the parallelism of the job. This
+change allows users to quickly upscale the job under backpressure.
+
+### SQL
+
+#### Officially deprecate the legacy planner
+
+##### [FLINK-21709](https://issues.apache.org/jira/browse/FLINK-21709)
+
+The old planner of the Table & SQL API is deprecated and will be dropped in Flink 1.14. This means
+that both the BatchTableEnvironment and DataSet API interop are reaching end of life. Use the
+unified TableEnvironment for batch and stream processing with the new planner, or the DataStream API
+in batch execution mode.
+
+#### Disable problematic cast conversion between NUMERIC type and TIMESTAMP type
+
+##### [FLINK-21698](https://issues.apache.org/jira/browse/FLINK-21698)
+
+The CAST operation between `NUMERIC` type and `TIMESTAMP` type is problematic and is disabled now,
+e.g.
+`CAST(numeric AS TIMESTAMP(3))` is disabled and should use `TO_TIMESTAMP(FROM_UNIXTIME(numeric))`
+instead.
```

Comment:
Shouldn't we add more explanation of changed time functions ? I think this is the important breaking change too. Like `PROCTIME(), CURRENT_TIMESTAMP, CURRENT_TIME`, they're heavily used in Table/SQL jobs, but we didn't remind int he release note.

Reply:
Could you please fill in the release notes of the corresponding tasks and link them here? All of the points in this document are extracted from release notes of JIRA issues.

Reply:
sure